Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -545,6 +561,30 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
     private var _decelerationLastTime: TimeInterval = 0.0
     private var _decelerationDisplayLink: NSUIDisplayLink!
     private var _decelerationVelocity = CGPoint()
+
+    @objc public func addOutsideGestureRecognizer(recognizer: UIGestureRecognizer) {
```

Comment:
Minor, `addAdditionalGestureRecognizer` might relay the intent of this method more clearly (to avoid conflating `outside` with being outside the view 😅)